packages/@okta/vuepress-site/docs/guides/add-an-external-idp/configure-idp-in-okta/openidconnect/appidpinokta.md

```diff
@@ -0,0 +1,20 @@
+In the **General Settings** section:
+
+* **Name**: Enter the name that you would expect to see on a button, such as **Sign in with MyIdentityProvider**.
+* **Client Id**: Paste the app ID or client ID that you obtained when you configured the Identity Provider in the <GuideLink link="../create-an-app-at-idp">previous section</GuideLink>.
+* **Client Secret**: Paste the secret that you obtained in the <GuideLink link="../create-an-app-at-idp">previous section</GuideLink>. 
+* **Scopes**: Leave the defaults. These scopes are included when Okta makes an OpenID Connect request to the Identity Provider.
+
+    By default, Okta requires the `email` attribute for a user. The `email` scope is required to create and link the user to Okta's Universal Directory. If your Identity Provider doesn't support this attribute, you can make it optional. See [Manage User Profiles](https://help.okta.com/en/prod/Content/Topics/Directory/eu-profile-editor.htm).
+
+In the **Endpoints** section:
+
+Add the following endpoint URIs for the OpenID Connect Identity Provider that you are configuring. You can find the endpoints in the well-known configuration document for the Identity Provider (for example, `https://{theIdPdomain}/.well-known/openid-configuration`. 
+
+For a list of fully-tested and supported Identity Providers, see [OpenID Connect Identity Providers](/docs/guides/add-an-external-idp/before-you-begin/#supported-openid-connect-identity-providers).
+
+* **Issuer** - The identifier of the OpenID Connect provider. For example, where you created the Identity Provider app: `https://{theIdPdomain}`
+* **Authorization endpoint** - The URL of the Identity Provider's OAuth 2.0 Authorization endpoint. For example: `https://{theIdPdomain}/oauth2/v1/authorize`
+* **Token endpoint** - The URL of the Identity Provider's token endpoint for obtaining access and ID tokens. For example: `https://{theIdPdomain}/oauth2/v1/token`
+* **JWKS endpoint** - The URL of the Identity Provider's JSON Web Key Set document. This document contains signing keys that are used to validate the signatures from the provider. For example: `https://{theIdPdomain}/oauth2/v1/keys`
+* **Userinfo endpoint** - The endpoint for getting identity information about the user. For example: `https://{theIdPdomain}/oauth2/v1/userinfo`
```

Comment:
In the world today (where we just have a generic OIDC guide, and don't have guides for providers such as PayPal and Yahoo), I still think it makes sense to move the `Supported OpenID Connect Identity Providers` content currently on `Before you begin` to here.

The user needs that info – the well-known URIs and details for generic OIDC providers – here, not before they begin. This info is just for generic OIDC, and isn't relevant to other providers.